Wikipedia/Code/ImageDimmingExampleViewController.swift

```diff
@@ -20,6 +20,6 @@ extension ImageDimmingExampleViewController: Themeable {
         }
         
         view.backgroundColor = theme.colors.baseBackground
-        view.alpha = theme.imageOpacity
+        view.subviews.first?.alpha = theme.imageOpacity
```

Comment:
dimming the entire view looks bad

![dimmingview](https://user-images.githubusercontent.com/9299317/28902260-d23951d0-77cb-11e7-8150-c4abf97b8276.PNG)